docs/data.md

```diff
@@ -53,6 +53,11 @@ When an interactive plot is displayed in the dashboard the user is able to hover
 ![Plotly Plots](img/plotly.png)
 
 ### Movies
-For static images a movie of all of the available time steps can be downloaded by right-clicking on either the plot itself or the item name in the naviagation menu.
+A movie of all of the available time steps can be downloaded by right-clicking on the plot itself.
 
 ![Movie Download](img/movie.png)
+
+### Global Data Range
+For dynamic plots right-clicking on the plot will also display a second option: `Set global range for {current parameter}`. Selecting this option will allow the user to set the y-axis range for all timesteps for the selected parameter. Selecting `Clear` and then saving the range will reset the axis to the default behavior of resizing based on the current timestep range.
```

Comment:
Need to update here as well.

Reply:
Sorry about that, fixed here and [here](https://github.com/Kitware/eSimMon/pull/113/commits/e3d67acaddce7454d0b1bf3354567eb9e5f98146#diff-fc60afbe402c5dcfad4d76eeae53ddc6a2210963338b65c882cf6167f2b51a29R28) as well now.